src/iexamine.cpp

```diff
@@ -288,13 +293,25 @@ bool iexamine::harvestable_now( const tripoint &examp )
 void iexamine::cvdmachine( Character &you, const tripoint & )
 {
     // Select an item to which it is possible to apply a diamond coating
-    item_location loc = g->inv_map_splice( []( const item & e ) {
-        return ( e.is_melee( damage_type::CUT ) || e.is_melee( damage_type::STAB ) ) &&
-               e.made_of( material_steel ) &&
-               !e.has_flag( flag_DIAMOND ) && !e.has_flag( flag_NO_CVD );
+    std::array<std::reference_wrapper<const material_id>, 6> steels = {
+        material_steel,
```

Comment:
While you're at it, I think iron could work here as well.

Reply:
I'm working towards removing any crafted iron weapons, by the time a player can work iron it's hardly any more difficult to make steel and the result is more worth the CVD process.